sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormField.cs

```diff
@@ -39,13 +39,13 @@ internal FormField(string name, FieldValue_internal fieldValue, IReadOnlyList<Re
             Name = name;
             LabelData = null;
 
-            // Bounding box, page and text are not returned by the service in two scenarios:
```

Comment:
I couldn't find a reason why `text` was required here, so after looking at the output from the service, our existing code, and running recordings, it is safe to remove.

Reply:
So there's no scenario in which we'll have no bounding box, no page, but a NON-NULL text, right?

Reply:
no bounding box, no page, and yes text.
A user sent us a document where this was true :| first time seeing it too.
See service output:
```
"type": "object",
     "valueObject": {
         "CountryRegion": {
         "type": "countryRegion",
         "text": "ISL",
         "valueCountryRegion": "ISL"
     },
```

Reply:
We also have:
```
"type": "object",
     "valueObject": {
         "CountryRegion": {
         "type": "countryRegion",
         "valueCountryRegion": "ISL"
     },
```

And both are valid, so I see no point of having that `text` property there